graylog2-web-interface/src/components/common/PageNavigation.tsx

```diff
@@ -71,30 +75,39 @@ type Props = {
     path: string;
     permissions?: string | Array<string>;
     exactPathMatch?: boolean;
+    useIsValidLicense?: () => boolean;
+    position?: PluginNavigation['position'];
   }>;
 };
 
 /**
  * Simple tab navigation to allow navigating to subareas of a page.
  */
-const PageNavigation = ({ items }: Props) => (
-  <Container>
-    {items.map(({ path, title, permissions, exactPathMatch }) => {
-      if (!path) {
-        return null;
-      }
+const PageNavigation = ({ items }: Props) => {
+  const currentUser = useCurrentUser();
 
-      return (
-        <IfPermitted permissions={permissions ?? []} key={path}>
-          <LinkContainer to={path} relativeActive={!exactPathMatch}>
-            <StyledButton bsStyle="transparent">
-              <NavItemStateIndicator>{title}</NavItemStateIndicator>
-            </StyledButton>
-          </LinkContainer>
-        </IfPermitted>
-      );
-    })}
-  </Container>
-);
+  const formatedItems = useMemo(() => {
+    const availableItems = items.filter(
+      (item) =>
+        (typeof item.useIsValidLicense === 'function' ? item.useIsValidLicense() : true) &&
+        isPermitted(currentUser.permissions, item.permissions) &&
+        !!item.path,
+    );
+
+    return sortNavigationItems(availableItems, 'title');
```

Comment:
In a follow-up PR I would like to unify the title / description attribute of the main navigation and the page navigation.

Reply:
I addressed this in https://github.com/Graylog2/graylog2-server/pull/22671